tests/behat/features/entity_clone.feature

```diff
@@ -0,0 +1,48 @@
+@suggest
```

Comment:
Great to see a test added. I think it can be simplified to improve readability and scope.

For instance there's no need to test for published or moderation state so they're not required in the test content setup step.

Other superfluous steps:
```
    Then save screenshot
    Then I should see an "input#edit-field-node-site-10010" element
    And I should see an "input#edit-field-node-site-10011" element
    And I should see an "input#edit-field-node-site-10014" element
    And I fill in "Title" with "Bibendum Pharetra Inceptos"
    And I fill in "Summary" with "Cras Tristique Risus"
    And I fill in "Topic" with "Test topic 1 (10017)"
```

Also have a look at the test in [tide_core for entity_clone](https://github.com/dpc-sdp/tide_core/pull/193/files#diff-069f003f35ffbc3a4f34940b173d9b3d3b868f23604275e0f3dd1c809ce7b41eR18) for an example of navigating to the node edit form. It improves readability to follow this approach.

The test would need to be updated to check for the presence of the clone option on the node edit form and should also be extended to check on the content admin screen.

Reply:
thanks @GROwen , will update the tests.